### DISCUSSION TITLE
Where do you initialize/set the suggestions (next0, next1, ...)?

### DISCUSSION THREAD

**Discussion by 20170010** [florisboard/florisboard#2115] [Q&A]
I can't find where you initialize/set the suggestions in the suggestion bar? Respectively, where do you generelly use the input you get from the keyboard and show it in the suggestion bar? If you type for example "test", the suggestion bar spits out "test0, test1, test2, ...", where do you proccess the given data in generel? Thanks in advance!

Picture for refference what i mean with "next0, next1, ..."

![image](https://user-images.githubusercontent.com/112947872/202922370-7e5affc8-2c11-471b-a6ce-54daaef2526a.png)

**Comment by moonbeamcelery** [comment]
[These](https://github.com/florisboard/florisboard/blob/470d302ed2fbde17e600dd6d90ce696460b85233/app/src/main/kotlin/dev/patrickgold/florisboard/ime/nlp/NlpManager.kt#L173), which ultimately calls [these](https://github.com/florisboard/florisboard/blob/4c800f37f09f7e6436414045e111df239ba1c5f3/app/src/main/kotlin/dev/patrickgold/florisboard/ime/nlp/latin/LatinLanguageProvider.kt#L102).
